airflow/providers/qdrant/provider.yaml

```diff
@@ -0,0 +1,54 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-qdrant
+
+name: Qdrant
+
+description: |
+    `Qdrant <https://qdrant.tech/documentation>`__
+
+state: not-ready
```

Comment:
I think you shoudl change it to `ready` - we use not-ready only when we do not want to release provider **just yet** but this one seems quite, well , ready to be released.

Reply:
https://github.com/apache/airflow/pull/36805/commits/cdf713ee0e71a05d8e263b0d0f04ce02657e44e9